registries/_format/decimal.md

```diff
@@ -0,0 +1,21 @@
+---
+owner: baywet
+issue:
+description: A fixed point decimal number as defined by ISO/IEC 9075-2 2016 12 15
+base_type: number
```

Comment:
I'm not sure it's viable to implement decimal as a JSON number, because pre-decimal leading and post-decimal trailing zeros are invisible in the JSON Schema data model.  The typical way to implement reliable fixed point decimal is to represent it as a JSON string.

Reply:
Can you expand a little on this comment please?

are you referring to the fact that 
```json
{
	"balance": -1.2300E+4,
	"volume": 00010.2000
}
```

is not valid JSON because of the leading zeros?

Reply:
I forgot that leading zeros aren't valid, so please ignore that part.

What I mean is that there is no way to distinguish between:
```JSON
{
    "volume": 0.2000
}
```
and
```JSON
{
    "volume": 0.2
```

in JSON Schema.  Or really, in the vast majority of parsed JSON representations, because the data types into which those values are parsed work with the mathematical values and not the lexicographic representations.

So if your schema is:
```JSON
{
    "type": "number",
    "format": "decimal"
}
```

What does that mean here?  Are both of the above examples valid?  Should users expect the top one to indicate that the fixed with is 4 decimal places and the bottom one as 1 decimal place?  JSON Schema cannot make that distinction.  To it, both of those represent the mathematical number `0.2`.

Reply:
and we'd end up loosing precision in the process making scientists mad at us. Thanks for calling this out.
```suggestion
base_type: string
```

Reply:
Yup, exactly.  I believe if you look at how financial standards handle data interchange in JSON, they always use strings for numbers to both keep precision and avoid floating point errors of all sorts.